src/VAP/VideoPipelineCore/VideoPipelineCore.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenCvSharp4" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.ubuntu.18.04-x64" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.win" Version="4.0.0.20190108" />
+    <PackageReference Include="OpenCvSharp4" Version="4.1.1.20191110" />
+    <PackageReference Include="OpenCvSharp4.runtime.ubuntu.18.04-x64" Version="4.1.1.20191023" />
+    <PackageReference Include="OpenCvSharp4.runtime.win" Version="4.1.1.20191110" />
```

Comment:
📝 I'm not sure why there is a runtime mismatch between ubuntu and win, but it appears these are not always uploaded to NuGet as a pair.